bucket/retroarch-nightly.json

```diff
@@ -0,0 +1,69 @@
+{
+    "homepage": "https://www.retroarch.com/",
+    "description": "Frontend for emulators, game engines and media players (nightly)",
+    "license": "GPL-3.0-only",
+    "version": "2022-02-12",
+    "architecture": {
+        "64bit": {
+            "url": "https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch.7z",
+            "hash": "20333f4eec379cb7ba30a5b8bb781aea70708c7381905e2ba0495221b0c2030c",
+            "extract_dir": "RetroArch-Win64"
+        },
+        "32bit": {
+            "url": "https://buildbot.libretro.com/nightly/windows/x86/RetroArch.7z",
+            "hash": "6bb2e6b12d47505117baa033f5f654e4dd65c44afa21e18f61c01da9f8a92371",
+            "extract_dir": "RetroArch-Win32"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\retroarch.cfg\")) {",
+        "    New-Item \"$persist_dir\\retroarch.cfg\" -Type File | Out-Null",
+        "}"
+    ],
+    "bin": [
+        [
+            "retroarch.exe",
+            "retroarch"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "retroarch.exe",
+            "RetroArch"
+        ]
+    ],
+    "persist": [
+        "assets\\wallpapers",
+        "cores",
+        "cheats",
+        "overlays",
+        "playlists",
+        "records",
+        "recordings",
+        "saves",
+        "screenshots",
+        "states",
+        "system",
+        "thumbnails",
+        "retroarch.cfg"
+    ],
+    "checkver": {
+        "url": "http://worldtimeapi.org/api/timezone/America/Los_Angeles",
+        "regex": "utc_datetime\":\"(\\d{4}-\\d{2}-\\d{2})"
+    },
```

Comment:
Since this won't be 100% in sync with RetorArch nightly updates, I think it's better to drop the `hash` and `checkver` entries and set the version identifier to `nightly`. This skips hash verification entirely and prevents installations from failing due to mismatching hashes.